### ISSUE TITLE
White rim to skin lesion

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9999]
**Preferred term label:**
White rim to skin lesion

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Keratoacanthoma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
if this is a characteristic that only belongs to Keratoacanthoma, it would be better to include it in the definition of that term?

**Comment by MickeySegal** [comment]
We only have this finding listed so far for Keratoacanthoma so we can eliminate it.

**Comment by pnrobinson** [comment]
OK
